### ISSUE TITLE
Timer example on Windows redraws without clear, leading to very ugly text

### ISSUE THREAD

**Issue by chris-morgan** [andlabs/libui#453]
Not having investigated yet, I’m not sure if this is a bug in the timer example or in libui in general, but I am inclined to think the latter.

The issue is most readily demonstrated with a screenshot of running the timer example on Windows:

<img src=https://user-images.githubusercontent.com/392868/51586777-a6df7900-1f32-11e9-9fd5-bc7e7d1cc18d.png width=322 height=272 alt="Screenshot of running timer.exe, demonstrating the claimed bug">

This is the example that adds a new line each second. It appears to be redrawing each line without erasing it beforehand, and so since the text is antialiased, the end result is progressively increasing ugliness. The bottom line has been drawn once, the second-bottom twice, and so on.

I observed these symptoms in other cases too; this was just the most obvious example to demonstrate it in.

(The image may appear even wonkier than it should since my display is 2×.)

**Comment by andlabs** [comment]
Yeah, I'm not sure why this happens, though I'm sure someone here has pointed this out before...
